frontend/src/core/app/helpers.js

```diff
@@ -190,6 +186,14 @@ define(function(require){
 
         },
 
+        ifImageIsCourseAsset: function(url, block) {
```

Comment:
Possibly those functions need named better too - the main problem I have is that the the function name looks like it tests something, but actually modifies the arguments.